src/commands/sbt_cached.yml

```diff
@@ -17,6 +17,10 @@ parameters:
     description: "The prefix of cache to be used"
     type: string
     default: sbt-cache-032020
+  store_test_results:
+    description: "Whether to upload the test results back to CircleCI"
+    type: boolean
```

Comment:
I think that this can be slightly improved:
 - `store_test_results` -> `store_test_results_path`
 - `boolean` -> `string`
 - `default: ""`

Keep the condition as it is (just update the parameter name) and:
```
        - store_test_results:
            path: ~/workdir
```
becomes:
```
        - store_test_results:
            path: << parameters.store_test_results_path >>
```

WDYT?

Reply:
Done 👍